apps/change-log.md

```diff
@@ -14,6 +14,16 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
+## 2018-10-11
```

Comment:
We will have merge conflicts here, because the same file is touched in your other PR. Best way is, to do the changelog in a separate PR, once there is more than one PR for a release.

Reply:
Same here for the date, as commented in the other PR. Depends a bit on when Infinity will do their part.